src/main/locales/da/bf_locale.h

```diff
@@ -0,0 +1,246 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// DK translations author: HThuren <thuren.henrik@gmail.com>
+
+#define STR_THOUSAND 	            "."
+#define STR_SECONDS                  "sekunder"
+
+// CMS
+#define CMS_STARTUP_HELP_TEXT1      "MENU:GAS MIDT"
+#define CMS_STARTUP_HELP_TEXT2      " + SIDEROR VENSTRE"
+#define CMS_STARTUP_HELP_TEXT3      " + HOEJDE OP"
+
+// OSD
+#define STR_OSD_ARMED                TR2("* AKTIV *", "*** AKTIVERET ***")
```

Comment:
My suggestion to make translation easier is to do the TR2(STR_OSD_ARMED_SD, STR_OSD_ARMED_HD) where it is to be used in code, and then define two separate here.

#define STR_OSD_ARMED_SD "* AKTIV *"
#define STR_OSD_ARMED_HD "*** AKTIVERET ***"

We can then have the translation request output generation know the max length of STR_OSD_ARMED_SD STR_OSD_ARMED_HD for the translators.

Reply:
Like
#define STR_OSD_ARMED_SD "* AKTIV *"
#define STR_OSD_ARMED_HD "*** AKTIVERET ***"
And
#define STR_OSD_ARMED                TR2( STR_OSD_ARMED_SD, STR_OSD_ARMED_HD)
or #ifdef xxx #endif in code

Reply:
Only issue here are, that it differs a lot when need to use TR2 system. If placed in code, we always need the 2 defines, maybe equael and somtimes differ. From EdgeTx translation I see CZ and IT often have issues with long words, but other languages don't have same need.

Reply:
Good point on the both needed...

Let's see if we can hook to crowdin with what you have.